AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/AnnotationHelper.java

```diff
@@ -55,6 +66,40 @@
 
 	public static final String DEFAULT_FIELD_NAME_VALUE = "value";
 	public static final String DEFAULT_FIELD_NAME_RESNAME = "resName";
+	private static final String PARCEL_ANNOTATION = "org.parceler.Parcel";
```

Comment:
Can you move all Parceler-related things to another helper class (which can extend `AnnotationHelper`)?